app/src/main/java/ru/suslovalex/androidacademyapp/FragmentMoviesList.kt

```diff
@@ -21,25 +19,11 @@ class FragmentMoviesList: Fragment() {
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
-
         view.findViewById<View>(R.id.item).setOnClickListener {
-            itemListener?.moveToMovieDetails()
-        }
-    }
-
-    override fun onAttach(context: Context) {
-        super.onAttach(context)
-        if (context is ClickItemListener){
-            itemListener = context
+            activity?.supportFragmentManager
+                ?.beginTransaction()
+                ?.add(R.id.container_layout, FragmentMoviesDetails())
```

Comment:
@AleksandrAlex Please add `replace` here and back `add` when you creating `FragmentMoviesList`